src/vs/workbench/contrib/chat/browser/chat.contribution.ts

```diff
@@ -139,7 +150,11 @@ class ChatAccessibleViewContribution extends Disposable {
 				verbositySettingKey: 'panelChat',
 				provideContent(): string { return responseContent; },
 				onClose() {
-					widget.focus(focusedItem);
+					if (focusedInput) {
+						widget?.focusInput();
+					} else {
+						widget!.focus(focusedItem!);
```

Comment:
Was the `widget!` intentional? The else uses `?` and this could be undefined if `editor` is falsy

Reply:
if `widget` is undefined, then `focusedItem` will be undefined. and we return if that's true

Reply:
https://github.com/microsoft/vscode/blob/ae5d6a86630f4801aec3125cec08d7afc9c43f14/src/vs/workbench/contrib/chat/browser/chat.contribution.ts#L141

Reply:
they should both use `!`

Reply:
oh, I see what you mean actually

Reply:
this is confusing atm. i'll clean it up